internal/repositories/file.go

```diff
@@ -0,0 +1,57 @@
+package repositories
+
+import (
+	"encoding/json"
+	"github.com/avidyakov/shortener/internal/config"
+	"log"
+	"os"
+)
+
+type fileRepo struct {
+	links map[string]string
+}
+
+func NewFileRepo() LinkRepo {
+	content, err := os.ReadFile(config.Conf.File)
+	var links map[string]string
+
+	if err == nil {
+		err = json.Unmarshal(content, &links)
+		if err != nil {
+			log.Fatal(err)
+		}
+	} else {
+		links = make(map[string]string)
+	}
+
+	return &fileRepo{
+		links: links,
+	}
+}
+
+func (r *fileRepo) GetLink(shortLinkID string) (originLink string, ok bool) {
+	originLink, ok = r.links[shortLinkID]
+	return
+}
+
+func (r *fileRepo) CreateLink(shortLinkID string, originLink string) {
+	r.links[shortLinkID] = originLink
+	r.writeFile()
+}
+
+func (r *fileRepo) RemoveLink(shortLinkID string) {
+	delete(r.links, shortLinkID)
+	r.writeFile()
+}
+
+func (r *fileRepo) writeFile() {
+	jsonData, err := json.MarshalIndent(r.links, "", "  ")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = os.WriteFile(config.Conf.File, jsonData, 0644)
```

Comment:
как прекрасно было бв если выглядела запись примерно так
```suggestion
	err = os.WriteFile(r.pathFile, jsonData, 0644)
```